src/View/ViewBuilder.php

```diff
@@ -301,6 +301,18 @@ public function getPlugin(): ?string
         return $this->_plugin;
     }
 
+    /**
+     * Set a helper to use.
+     *
+     * @param string $helper Helper to use.
+     * @param bool $merge Whether or not to merge existing data with the new data.
+     * @return $this
+     */
+    public function setHelper(string $helper, bool $merge = true)
+    {
+        return $this->setHelpers([$helper], $merge);
```

Comment:
I wish these weren't named `set` because that implies overwriting when it is adding by default.

Reply:
Yeah, but then you would need two methods. add* and set* :)

Reply:
Why would we need add* and set*?  We could just have `addHelper(string $helpers)`. Requiring to overrite existing helpers list is pretty rare in which case one case use `setHelper()` which has `$overwrite` argument.

Reply:
Fair enough.

Reply:
Wouldnt then deprecate the merge flag be cleaner?
And
- addHelper()
- addHelpers()
- setHelpers()

which each only 1 arg?